app/components/visualizer/main-viz/main-viz.jsx

```diff
@@ -0,0 +1,3 @@
+import Chart from "./templates/Chart.jsx"
```

Comment:
Since only index.js is sourced by the compiler, everything else must be imported. This is common in most (all?) languages actually, and keeps the global namespace clean. The alternative would mean everything gets imported everywhere, which when mixed with JS scoping sounds like... the opposite of fun.